lib/test_web/templates/page/index.html.heex

```diff
@@ -0,0 +1,5 @@
+<h1> Welcome to Exzeitable!</h1>
+
+<p> Here are the examplary (exzei-)tables: </p>
+<li><span><%= link "Posts", to: Routes.post_path(@conn, :index) %></span></li> 
+<li><span><%= link "Users", to: Routes.user_path(@conn, :index) %></span></li> 
```

Comment:
Do these need to go in a `<ul>` tag?

Reply:
It works this way. But it might not be THE way.
So I will add ul tag